lexicon/atrium-codegen/src/token_stream.rs

```diff
@@ -556,7 +555,8 @@ fn string_type(string: &LexString) -> Result<(TokenStream, TokenStream)> {
         Some(LexStringFormat::Handle) => quote!(crate::types::string::Handle),
         Some(LexStringFormat::Nsid) => quote!(crate::types::string::Nsid),
         Some(LexStringFormat::Language) => quote!(crate::types::string::Language),
-        // TODO: other formats
```

Comment:
`at-uri` and `uri` are still not parsed here yet, so I think this `TODO` still applies (unless we specifically don't want to parse them here).